mPower2/MotorControl/Resources/Walk_And_Balance.json

```diff
@@ -159,45 +159,39 @@
                      }
                  }
              },
+             {
```

Comment:
Are we ok with not having async results on the countdown?

Reply:
Yes, I think that we are.  ;)

If researchers really want to break up the results into "walk" and "balance" then it really doesn't make sense to include the countdown for the "walk" but **not** for the "balance".